.travis.linux.trusty.sh

```diff
@@ -0,0 +1,21 @@
+#! /bin/bash
+
+URL=ftp://ftp.simplesystems.org/pub/libpng/png/src/libpng16/libpng-1.6.23.tar.gz
+FILE=libpng-1.6.23.tar.gz
```

Comment:
libpng is a pretty standard stable library. There are lots of ways people can install it.

A more appropriate solution is to add an alternate code path for macOS which uses a built in library for reading PNG files, and maybe PDF too!

I also like Brian's idea to make a Homebrew formula, similarly we can make Ubuntu packages, and maybe even a snap package. None of these things require adding a submodule.